src/vs/workbench/api/common/extHostExtensionService.ts

```diff
@@ -653,7 +653,8 @@ export abstract class AbstractExtHostExtensionService extends Disposable impleme
 		this._register(this._extHostWorkspace.onDidChangeWorkspace((e) => this._handleWorkspaceContainsEagerExtensions(e.added)));
 		const folders = this._extHostWorkspace.workspace ? this._extHostWorkspace.workspace.folders : [];
 		const workspaceContainsActivation = this._handleWorkspaceContainsEagerExtensions(folders);
-		const eagerExtensionsActivation = Promise.all([starActivation, workspaceContainsActivation]).then(() => { });
+		const remoteResolverActivation = this._handleRemoteResolverEagerExtensions();
```

Comment:
Could we move this to be the first line in the file? Maybe that would trigger the activation process for these extensions a bit earlier than the `*` ones.

Reply:
Is the intention for this to serve as the activation mechanism when the resolver is involved in setting up the remote connection as well? If it's just in the case where the resolver is not involved, I'm not sure it deserves higher priority.